libcodechecker/analyze/analysis_manager.py

```diff
@@ -187,7 +200,7 @@ def check(check_data):
     skiplist handler is None if no skip file was configured.
     """
 
-    action, context, analyzer_config_map, \
+    actions_map, action, context, analyzer_config_map, \
```

Comment:
Could we find out a better name for this. `actions_map` does not contain any compilation actions, only extra information for them.

Reply:
Another comment do we have to forward the whole dictionary for each of the processes in the process pool? Can we forward only the data with belongs to the build action handled by the current process?

Reply:
Actually, `actions_map` does contain all the build actions indexed by a (source, target) pair.
Unfortunately we need this map for all processes in case of CTU because we get the build action for a referenced TU from this map in case of the failure.